opengrok-indexer/src/test/java/org/opengrok/indexer/util/StringUtilsTest.java

```diff
@@ -39,12 +39,12 @@ public void testValues() {
         int i;
         long[] values = {
             0, 100, 1000, 1500, 64000, 124531, 3651782, 86400000, 86434349,
-            1075634299
+            1075634299, 86480001
         };
         String[] expected = {
-            "0", "100 ms", "1.0 seconds", "1.500 seconds", "0:01:04",
-            "0:02:04", "1:00:51", "1 day", "1 day 34.349 seconds",
-            "12 days 10:47:14"
+            "0 ms", "100 ms", "1.0 seconds", "1.500 seconds", "0:01:04",
+            "0:02:04", "1:00:51", "1 day", "1 day",
+            "12 days 10:47:14", "1 day 00:01:20"
```

Comment:
could you add a `86400001` (one day + one ms) too?

Reply:
OK